### PR TITLE
WIP: Improved project layout

### PR DESCRIPTION
**Pull request by havish10**

### CODE REVIEW

File: components/Project.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+export default class extends React.Component {
+  state = {
+    hover: false,
```
Comment (danielhuang) [inline components/Project.jsx]: This will only support a single image without additional state entries.

File: components/Project.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  render() {
+    return (
+      <div>
+        <img
+          src="https://cdn.discordapp.com/attachments/583367531406950431/585948245461696531/GroupNews451Combined.png"
+          alt="News 451 website"
+          onMouseEnter={() => this.setState({ hover: true })}
```
Comment (danielhuang) [inline components/Project.jsx]: Use css `:hover`

**Comment by havish10** [comment]
I still need to make the image smaller as they are 1.25mb each.

**Comment by danielhuang** [comment]
Netlify will compress it automatically.

File: components/Projects.jsx [hunk 3]
```diff
@@ -1,35 +1,37 @@
...
-      recent news without clutter or ads. Users can gauge reliability and bias
-      of their news using our custom machine learning model.
-    </p>
-    <img
-      src="https://cdn.discordapp.com/attachments/583367531406950431/585948245461696531/GroupNews451Combined.png"
-      alt="News 451 website"
+  <div style={{ paddingTop: '15px' }}>
+    <Project
+      name="News451"
+      img="https://cdn.discordapp.com/attachments/583367531406950431/591331216989356063/news451big.png"
```
Comment (danielhuang) [inline components/Projects.jsx]: Images should be hosted on the site itself.

File: components/Projects.jsx [hunk 4]
```diff
@@ -1,35 +1,37 @@
...
+      icon="https://pbs.twimg.com/profile_images/839515548333420545/ThVZeSAJ_400x400.jpg"
+      prizeDescription="1st place at the Sixth Baltimore Hackathon"
+      prize="$1000"
+    />
+    <hr
+      style={{
+        marginLeft: '-50vw',
+        marginRight: '-50vw',
+      }}
+    />
```
Comment (danielhuang) [inline components/Projects.jsx]: This actually creates 2 horizontal lines:
image

File: components/Project.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    <p>{props.description}</p>
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        paddingTop: '50px',
+      }}
+    >
+      <img
+        alt="bmore"
```
Comment (danielhuang) [inline components/Project.jsx]: What is this?

File: components/Projects.jsx [hunk 6]
```diff
@@ -1,35 +1,37 @@
...
-      News 451 is a minimalistic web application that allows users to see the
-      recent news without clutter or ads. Users can gauge reliability and bias
-      of their news using our custom machine learning model.
-    </p>
-    <img
-      src="https://cdn.discordapp.com/attachments/583367531406950431/585948245461696531/GroupNews451Combined.png"
-      alt="News 451 website"
+  <div style={{ paddingTop: '15px' }}>
+    <Project
+      name="News451"
```
Comment (danielhuang) [inline components/Projects.jsx]: News 451 has a space in its name.

File: components/Project.jsx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    >
+      <h1>{props.name}</h1>
+      <img alt="News451" src={props.img} />
+    </div>
+    <p>{props.description}</p>
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        paddingTop: '50px',
```
Comment (danielhuang) [inline components/Project.jsx]: Avoid using `paddingTop` for spacing.

File: components/Project.jsx [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+        alt="bmore"
+        src={props.icon}
+        style={{ width: '100px', height: '100px', borderRadius: '50%' }}
+      />
+
+      <div style={{ paddingLeft: '50px' }}>
+        <h3 style={{ fontSize: '1.625em' }}>
+          <b>Awarded</b>
+        </h3>
+        <p style={{ marginTop: '10px' }}>
```
Comment (danielhuang) [inline components/Project.jsx]: Avoid using `marginTop` for spacing.

File: components/Project.jsx [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+        </p>
+      </div>
+    </div>
+    <style jsx>
+      {`
+        h1 {
+          font-size: 3.5em;
+        }
+        h3 {
+          margin: 0px 0px 0px 0px;
```
Comment (danielhuang) [inline components/Project.jsx]: Can be simplified to `margin: 0`.

File: package.json [hunk 10]
```diff
@@ -14,7 +14,6 @@
     "react": "^16.8.6",
     "react-bootstrap": "^1.0.0-beta.8",
     "react-dom": "^16.8.6",
-    "react-grid-system": "^4.4.6",
```
Comment (danielhuang) [inline package.json]: This is outside the scope of this PR.

File: components/Intro.jsx [hunk 11]
```diff
@@ -1,7 +1,7 @@
 import Names from './Names'
 
 export default () => (
-  <div>
+  <div style={{}}>
```
[the remark this answers is not in the document]
Comment (havish10) [reply components/Intro.jsx]: fixed

File: components/Projects.jsx (hunk 4 above)
Comment (havish10) [reply components/Projects.jsx]: Wow! I did not notice that on my 1080p monitor it looks like one line.

File: components/Project.jsx [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+          <b>Awarded</b>
+        </h3>
+        <p style={{ marginTop: '10px' }}>
+          {props.prizeDescription} <br />({props.prize})
+        </p>
+      </div>
+    </div>
+    <style jsx>
+      {`
+        @media screen and (max-width: 1200px) {
```
Comment (danielhuang) [inline components/Project.jsx]: `@media` queries are hard to maintain and unnecessary.

Comment (havish10) [reply components/Project.jsx]: It is necessary to change the flex direction to stack the icon and the award on top of each other.

File: components/Project.jsx [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+      <img alt="Project" src={props.img} />
+    </div>
+    <p>{props.description}</p>
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        paddingTop: '50px',
+      }}
+      id="container"
```
Comment (danielhuang) [inline components/Project.jsx]: Avoid using `id`

File: components/Project.jsx [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+    >
+      <img
+        alt="Icon"
+        src={props.icon}
+        style={{ width: '100px', height: '100px', borderRadius: '50%' }}
+        id="logo"
+      />
+
+      <div style={{ padding: '0px 50px 0px 50px' }} id="award">
```
Comment (danielhuang) [inline components/Project.jsx]: Avoid using `id`

File: components/Project.jsx [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+        justifyContent: 'center',
+        paddingTop: '50px',
+      }}
+      id="container"
+    >
+      <img
+        alt="Icon"
+        src={props.icon}
+        style={{ width: '100px', height: '100px', borderRadius: '50%' }}
+        id="logo"
```
Comment (danielhuang) [inline components/Project.jsx]: Avoid using `id`

File: components/Project.jsx (hunk 13 above)
Comment (danielhuang) [inline components/Project.jsx]: Avoid using `id`
